src/Setup.ts

```diff
@@ -0,0 +1,46 @@
+// This file is run before main.js to setup the tools that the action depends on
+// https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runspre
+
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+export const sqlcmdToolName = 'go-sqlcmd';
+export const sqlcmdVersion = '0.8.1';
```

Comment:
out of curiosity, is there any reason why we're hardcoding this specific version?

Reply:
This was the latest release of [go-sqlcmd](https://github.com/microsoft/go-sqlcmd). We can just update this in the future to a newer version.

Reply:
Sure. Not a huge deal at all, just more curious than anything 😄 . 

Perhaps there could be a 'latest' release that we could leverage from go-sqlcmd that always pointed to the latest bits.

Certainly not a blocker, just thinking out loud.

Reply:
[https://github.com/microsoft/go-sqlcmd/releases/latest](https://github.com/microsoft/go-sqlcmd/releases/latest) will redirect to the release with the newest tag. Maybe you can somehow grab the tag from the response to that URL before going to downloads.

That said maybe we should consider some custom tagging in the go-sqlcmd repo to identify the right build for this action to use just in case "latest" has a regression so we can test with latest before tagging it.

Reply:
Agreed, would be preferable to not need to update this each time go-sqlcmd has a release.

Reply:
I'll add support for this in a separate PR. Added #112 to track this.